pkg/sqlutil/pg/pg.go

```diff
@@ -0,0 +1,35 @@
+package pg
+
+import (
+	"os"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+	"github.com/scylladb/go-reflectx"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func NewSqlxDB(t testing.TB, dbURL string) *sqlx.DB {
+	err := RegisterTxDb(dbURL)
+	if err != nil {
+		t.Fatalf("failed to register txdb dialect: %s", err.Error())
+		return nil
+	}
+	db, err := sqlx.Open(string(TransactionWrappedPostgres), uuid.New().String())
+	require.NoError(t, err)
+	t.Cleanup(func() { assert.NoError(t, db.Close()) })
+	db.MapperFunc(reflectx.CamelToSnakeASCII)
+
+	return db
+}
+
+func DbUrlOrInMemory(t testing.TB) string {
```

Comment:
I think the go naming convention here is unfortunately caps. But maybe the package provides enough context to drop DB. And if we say Test, then the in memory part can be documented instead 🤷
```suggestion
func TestURL(t testing.TB) string {
```